src/ImmersedBoundaries/partial_cell_bottom.jl

```diff
@@ -144,43 +145,34 @@ Criterion is zb ≥ z - ϵ Δz
     return z★ < zb
 end
 
-@inline function bottom_cell(i, j, k, ibg::PCBIBG)
-    grid = ibg.underlying_grid
-    ib = ibg.immersed_boundary
-    # This one's not immersed, but the next one down is
-    return !immersed_cell(i, j, k, grid, ib) & immersed_cell(i, j, k-1, grid, ib)
-end
-
 @inline function Δrᶜᶜᶜ(i, j, k, ibg::PCBIBG)
     underlying_grid = ibg.underlying_grid
     ib = ibg.immersed_boundary
 
     # Get node at face above and defining nodes on c,c,f
-    z = rnode(i, j, k+1, underlying_grid, c, c, f)
+    rᶜᶜᶠₖ₊₁ = rnode(i, j, k+1, underlying_grid, c, c, f)
 
     # Get bottom z-coordinate and fractional Δz parameter
-    zb = @inbounds ib.bottom_height[i, j, 1]
+    rᵇᵒᵗᵗᵒᵐ = @inbounds ib.bottom_height[i, j, 1]
```

Comment:
a lot of unicode 😅 .
what about
```suggestion
    rb = @inbounds ib.bottom_height[i, j, 1]
```
or
```
r_bottom = @inbounds ib.bottom_height[i, j, 1]
```?

Reply:
I didn't go with `rb` because I thought I wasn't gonna be clear! But maybe it is :)

I'll change things.

Reply:
Went with `rᵇ` to both decrease unicode and stay in line with Oceananigans' standard to putting the location at a superscript. Let me know if that's okay!